factory/api/apiResolverFactory.go

```diff
@@ -490,7 +490,7 @@ func createBuiltinFuncs(
 	maxNumAddressesInTransferRole uint32,
 	dnsV2Addresses [][]byte,
 ) (vmcommon.BuiltInFunctionFactory, error) {
-	mapDNSV2Addresses := make(map[string]struct{})
+	mapDNSV2Addresses := make(map[string]struct{}, len(dnsV2Addresses))
```

Comment:
useless map length specification, here and on blockProcessorCreator

Reply:
changed, again, :| 
```
Map: An empty map is allocated with enough space to hold the
 specified number of elements. The size may be omitted, in which case
 a small starting size is allocated.
```
on other compilers, it would give you weak warnings.

Reply:
Yep, I missed the fact that it is not a slice here.